examples/electron-todo-list/public/electron.cjs

```diff
@@ -0,0 +1,29 @@
+const electron = require("electron");
+const path = require("path");
+
+const app = electron.app;
+const BrowserWindow = electron.BrowserWindow;
+
+try {
+  require('electron-reloader')(module);
+} catch (_) {}
+
+let mainWindow;
+
+function createWindow() {
+  // Create the browser window.
+  mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: { nodeIntegration: true, contextIsolation: false },
```

Comment:
Could we add a comment here why we need to disable `contextIsolation`?

Reply:
Done